frontend/lib/src/components/elements/ArrowTable/ArrowTable.test.tsx

```diff
@@ -52,4 +53,63 @@ describe("st._arrow_table", () => {
       screen.getByTestId("stTableStyledEmptyTableCell")
     ).toBeInTheDocument()
   })
+
+  it("renders with all borders when border=true", () => {
```

Comment:
Not sure if we need so many frontend tests here or if it's better to just have the e2e snapshot tests...

Reply:
My personal preference is to have e2e tests for style aspects. But probably fine to leave those in.